documentation/feature/medical-system.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: wiki
 title: Medical System
-description: ACE provide users with a more realistic medical system and comes in both a basic and advanced version. Both versions have overlap but each have their own unique characteristics. 
+description: ACE3 provide users with a more realistic medical system and comes in both a basic and advanced version. Both versions have overlap but each have their own unique characteristics.
 group: feature
 order: 4
 category: realism
 parent: wiki
 ---
-**Disclaimer:** The documentation for the medical system being extremely long it's highly advised to use the table of contents at the top right corner of the page.
+**Disclaimer:** As the documentation for the medical system being extremely long it's highly advised to use the table of contents at the top right corner of the page.
```

Comment:
"With the" instead of "As the"

or if you want to use "As the" then switch "being" with "is"